src/algebra/opposites.lean

```diff
@@ -470,3 +470,14 @@ def mul_equiv.op {α β} [has_mul α] [has_mul β] :
 /-- The 'unopposite' of an iso `αᵒᵖ ≃* βᵒᵖ`. Inverse to `mul_equiv.op`. -/
 @[simp] def mul_equiv.unop {α β} [has_mul α] [has_mul β] :
   (αᵒᵖ ≃* βᵒᵖ) ≃ (α ≃* β) := mul_equiv.op.symm
+
+section ext
+
+@[ext]
```

Comment:
```suggestion
/-- This ext lemma change equalities on `αᵒᵖ →+ β` to equalities on `α →+ β`.
This is useful because there are often ext lemmas for specific `α`s that will apply
to an equality of `α →+ β` such as `finsupp.add_hom_ext'`. -/
@[ext]
```